folium/features.py

```diff
@@ -232,7 +232,7 @@ class VegaLite(Element):
 
     def __init__(self, data, width=None, height=None,
```

Comment:
You're only updating `VegaLite` it seems, why not `Vega` also?

Reply:
good point, will do